modules/axhal/src/arch/aarch64/hv/exception.rs

```diff
@@ -1,51 +1,29 @@
 use core::arch::global_asm;
-use core::sync::atomic::{AtomicUsize, Ordering};
-
-use aarch64_cpu::registers::{ESR_EL2, FAR_EL2};
 use tock_registers::interfaces::*;
 
-use memory_addr::VirtAddr;
-use page_table_entry::MappingFlags;
-
-use super::exception_utils::*;
+global_asm!(include_str!("exception.S"));
```

Comment:
put this after all imports.

rename to trap.rs and trap.S to align with original convention of arceos.